test/Distributed/Distributed.Grains/Streaming/CounterReportingGrain.cs

```diff
@@ -0,0 +1,55 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Distributed.GrainInterfaces.Streaming;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Orleans;
+using Orleans.Hosting;
+using Orleans.Providers.Streams.Generator;
+using Orleans.Streams;
+using Orleans.Streams.Core;
+
+namespace Distributed.Grains.Streaming
+{
+    public class CounterGrain : Grain, ICounterGrain
+    {
+        private readonly ReportingOptions _options;
+        private readonly List<IGrainWithCounter> _trackedGrains = new();
+
+        public CounterGrain(IOptions<ReportingOptions> options)
+        {
+            Debugger.Break();
```

Comment:
I think we should remove this Debugger.Break() - I recall that it can cause issues in some environments, but maybe I'm misremembering.